pulsar/internal/connection.go

```diff
@@ -287,8 +287,8 @@ func (c *connection) doHandshake() bool {
 	c.cnx.SetDeadline(time.Time{})
 
 	if cmd.Connected == nil {
-		c.log.Warnf("Failed to perform initial handshake - Expecting 'Connected' cmd, got '%s'",
-			cmd.Type)
+		c.log.Errorf("Failed to perform initial handshake - Expecting 'Connected' cmd, got '%s'",
```

Comment:
About log level I think `Warnf` is more appropriate, it has not reached the Error level

Reply:
How do we define the error level ? The client can't not working when this error happend and why just got warn log?

Reply:
The message itself need to be changed though. It was basically a "debug" message. In this case, the user doesn't know what's a `Connected cmd`.